carml-rdf-mapper/src/test/resources/com/taxonic/carml/rdf_mapper/impl/Person.ld.json

```diff
@@ -0,0 +1,6875 @@
+{
```

Comment:
Mmm is er een alternatief voor het includen van dit enorme bestand?

Reply:
Ja goed punt, zal dat aanpassen